### DISCUSSION TITLE
Công cụ sửa lỗi bluetooth

### DISCUSSION THREAD

**Discussion by izasofa** [pembem22/jbl-firmware#9] [General]
Hi, I found the file you uploaded very helpful.
I want to ask a tool called: MPflashtool 3.0 (Mass product Flash Tool) by ZGmicro. I want to rerun my jbl charge 5 speaker. Thank you

**Comment by gogi-ex777** [comment]
I need too , do you find that Zgmicro tool?

**Comment by Hoanghoa113** [comment]
có vẻ việc tìm kiếm công cụ khá khó khăn tôi cũng đang tìm kiếm nó

**Comment by rethlang** [comment]
any one have this tool for sell? please contact me?

**Reply by traibanhnho** [reply]
toi co

**Reply by Subaruaho** [reply]
please

**Reply by rethlang** [reply]
> toi co

How to contact you?

**Reply by traibanhnho** [reply]
anh có dùng whatsapp không

**Reply by traibanhnho** [reply]
nhắn em số whatsapp

**Reply by rethlang** [reply]
My what app +855962737297

**Reply by bacmai221** [reply]
what app 0379016182

**Reply by traibanhnho** [reply]
> what app 0379016182
> 
> 

không thấy whatapp

**Comment by anhphuonghuyen1998-oss** [comment]


**Reply by traibanhnho** [reply]
tôi có
